src/index.js

```diff
@@ -0,0 +1,169 @@
+import { createProject, createTodo } from './create';
+
+let selectedProjectId1 = 1;
+let selectedTaskId1 = 1;
+
+const projectsContainer = document.querySelector('[data-projects]');
+const taskContainer = document.querySelector('[data-to-do]');
+const descriptionContainer = document.querySelector('[data-description]');
+
+const projectFormBody = document.getElementById('project_form_body');
+const taskFormBody = document.getElementById('task_form_body');
+// const newProjectForm = document.querySelector('.new_project');
+const newProjectInput = document.querySelector('[data-new-project-input]');
+// const newTaskInput1 = document.querySelector('[data-new-todo-form]');
+// const newTaskInput = document.getElementsByClassName('form-group');
+
+const newTodoInputTitle = document.querySelector('[data-new-todo-title-input]');
+const newTodoInputDesc = document.querySelector('[data-new-todo-desc-input]');
+const newTodoInputPrior = document.querySelector('[data-new-todo-prior-input]');
+const newTodoInputDate = document.querySelector('[data-new-todo-date-input]');
+const newTodoInputTime = document.querySelector('[data-new-todo-time-input]');
+const newTodoInputNote = document.querySelector('[data-new-todo-note-input]');
+
+const addProject = document.getElementById('add_project');
+const addTask = document.getElementById('add_task');
+
+const projectForm = document.getElementById('project_form');
+const taskForm = document.getElementById('task_form');
+
+const deleteTask = document.getElementById('button_delete');
+const updateTask = document.getElementById('button_update');
+
+const LOCAL_STORAGE_ALL_PROJECTS = 'projects';
+
+const projects = JSON.parse(localStorage.getItem(LOCAL_STORAGE_ALL_PROJECTS)) || [
+  {
+    id: Date.now().toString(),
+    name: 'Default Project',
+    todos: [],
+  },
+];
+
+const renderdescription = (selectedTaskId1, selectedProject) => {
+  descriptionContainer.innerHTML = '';
+  const selectedTask = selectedProject.todos.find((task) => task.id === selectedTaskId1);
+  descriptionContainer.innerHTML = `Name:- ${selectedTask.name} <br> Description:-${selectedTask.desc}
+        <br> Priority:- ${selectedTask.prior} <br> Date:- ${selectedTask.date} <br> note:- ${selectedTask.note} <br>
+         `;
+  deleteTask.style.visibility = 'visible';
+  updateTask.style.visibility = 'visible';
+};
+
+function description(task, selectedProject) {
+  const selectedTaskId1 = task.id;
+  renderdescription(selectedTaskId1, selectedProject);
+  return selectedTaskId1;
+}
```

Comment:
- [ ] Kindly separate this logic from the DOM by moving by placing it in a different file. Use import and export statements to use the logic in `index.js` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list))
- [ ] Kindly use the ES6 arrow function instead of the function keyword